mifospay/src/main/java/org/mifospay/notification/ui/NotificationScreen.kt

```diff
@@ -0,0 +1,185 @@
+    uiState: NotificationUiState,
+    isRefreshing: Boolean,
+    onRefresh: () -> Unit
+) {
+    val pullRefreshState = rememberPullRefreshState(isRefreshing, onRefresh)
+    Box(Modifier.pullRefresh(pullRefreshState)) {
+        Column(
+            modifier = Modifier.fillMaxSize()
+        ) {
+            MifosTopAppBar(titleRes = R.string.notifications)
+            when (uiState) {
+                is NotificationUiState.Error -> {
+                    EmptyContentScreen(
+                        modifier = Modifier,
+                        title = stringResource(id = R.string.error_oops),
+                        subTitle = stringResource(id = R.string.unexpected_error_subtitle),
+                        iconTint = Color.Black,
+                        iconImageVector = Icons.Rounded.Info
+                    )
+                }
+
+                NotificationUiState.Loading -> {
+                    MfLoadingWheel(
+                        contentDesc = stringResource(R.string.loading),
+                        backgroundColor = Color.White
+                    )
+                }
+
+                is NotificationUiState.Success -> {
+                    if (uiState.notificationList.isEmpty()) {
+                        Column(
+                            modifier = Modifier.fillMaxSize(),
+                            verticalArrangement = Arrangement.Center,
+                            horizontalAlignment = Alignment.CenterHorizontally
+                        ) {
+                            Text(
+                                text = stringResource(id = R.string.no_notification_found),
+                                style = styleMedium16sp
+                            )
+                        }
+                    } else {
+                        LazyColumn {
+                            items(uiState.notificationList) { notification ->
+                                NotificationList(
+                                    title = notification.title.toString(),
+                                    body = notification.body.toString(),
+                                    timestamp = notification.timestamp.toString()
+                                )
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        PullRefreshIndicator(
+            refreshing = isRefreshing,
+            state = pullRefreshState,
+            modifier = Modifier.align(Alignment.TopCenter)
+        )
+    }
+}
+
+@Composable
+fun NotificationList(title: String, body: String, timestamp: String) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp),
+        shape = RoundedCornerShape(12.dp)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(24.dp)
+        ) {
+            Text(
+                text = title,
+                style = styleMedium16sp,
```

Comment:
Use material theme `MaterialTheme.typography.bodySmall` 

`bodySmall` will be according to test styling. check in designsystem/MifosTypography which is perfect for the text